src/test/java/com/google/javascript/clutz/DeclarationSyntaxTest.java

```diff
@@ -29,31 +29,41 @@
       new FilenameFilter() {
         @Override
         public boolean accept(File dir, String name) {
-          return TS_SOURCES.accept(dir, name) && !name.contains("_emit_platform_externs");
+          return TS_SOURCES.accept(dir, name) && !name.contains("_with_platform");
         }
       };
 
   private static final FilenameFilter TS_SOURCES_WITH_PLATFORM_EXTERNS =
       new FilenameFilter() {
         @Override
         public boolean accept(File dir, String name) {
-          return TS_SOURCES.accept(dir, name) && name.contains("_emit_platform_externs");
+          // with incremental mode, _with_platform is going away, and a recent change makes our testing
+          // strategy for types_externs_with_platform not work, so special case it and remove when we
+          // stop doing _with_platform tests
+          return TS_SOURCES.accept(dir, name)
+              && name.contains("_with_platform")
+              && !name.contains("types_externs_with_platform");
```

Comment:
does that mean types_externs_with_platform is not tested? Then it should be deleted

Reply:
Added a separate test for it alone.